daemon/system-info.sh

```diff
@@ -105,6 +105,7 @@ else
                 CONTAINER_OS_DETECTION="/etc/os-release"
         fi
 
+        # shellcheck disable=SC2153
```

Comment:
if the condition on L103 fails you probably want to preinitialise these to `unknown`?

Reply:
I'm not sure. I'm just adding this because Shellcheck was complaining (incorrectly) about these being uninitialized (see the code just above) while an equivalent name in lowercase was (which is used near the bottom).